City-of-Helsinki/drupal-helfi-platform-config
# Events paragraph translations

# [Events paragraph translations UHF-7022](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7022)
Added needed translations for the event paragraph

## What was done
* Added paragraph translations
* Made changes to code to enable translations

## How to install
* Start up an instance of your choice
    * `composer require drupal/helfi_platform_config:dev-UHF-7022-events-translations`
    * `composer require drupal/hdbt:dev-UHF-3128-sote-linkedevents-filtering`
* `make shell` into the container
  * Run `drush locale:check && drush locale:update`
* Run `make drush-updb drush-cr`

## How to test
* Go to a landing or standard page. Add a paragraph by adding 'Events' paragraph to content / lower content region
  * Use this URL: https://tapahtumat.hel.fi?text=jooga for Api URL field
  * Toggle all the boolean field values to true
* Go to the created page. You should see list of events in the paragraph.
* Try using the paragraph with different language versions (by translating the node). The translated strings should match the ones in the files attached to the Jira ticket: https://helsinkisolutionoffice.atlassian.net/browse/UHF-7022
* **NOTE:** sketch has been updated since ordering the translations (some strings have been changed). Follow-up ticket for adding the changed translations has been created -> check that the strings are somewhat in line with what is defined in the sketch: https://share.goabstract.com/e4d7b8d2-b559-4a80-b2ff-432ee2ab83e4?collectionLayerId&mode
